src/graph/impl/KokkosGraph_Distance1Color_impl.hpp

```diff
@@ -2293,63 +2250,32 @@ class GraphColor_VB:public GraphColor <HandleType,lno_row_view_t_,lno_nnz_view_t
   };
 
 
-  /**
-   * Functor for parallel prefix sum
-   */
-  template <typename view_type>
-  struct parallel_prefix_sum{
```

Comment:
I thought this was used in EB too ? Does it ?

Reply:
@srajama1 VB and EB are slightly different now, but the single-pass scan idea is the same. VB is a scan over the current vertex worklist, and the scan populates the next worklist with vertices that were uncolored. These get marked in the current worklist by adding ``nv`` to them.

EB is a scan over the current edge worklist, and the next worklist is filled with edges where ``edge_conflict_marker(i)`` is nonzero.

Reply:
I am ok with two different scans.